mkposters/post_install.py

```diff
@@ -0,0 +1,53 @@
+import subprocess
+
+
+def main():
```

Comment:
Nit: can we just call this `post_install` rather than calling it different things in different files?

Reply:
Agreed, I'll rename.